pledge/index.html

```diff
@@ -127,6 +127,34 @@ <h1 class="ui inverted header">
     </div> <!-- End of card -->
 
 
+<div class="ui card">
+  <div class="content">
+    <img class="right floated mini ui image" 
+            src="https://avatars0.githubusercontent.com/u/37602">
+    <div class="header">
+      Nicholas H.Tollervey
+    </div>
+    <div class="description">
+      Computing, music, philosophy, teaching & writing. Just like this bio: 
```

Comment:
Oops... should be &amp; - I'm doing this in a browser. Not sure how to update it. ;-)